### PR TITLE
Expose Register Expires setting and if we Register at all.

### PR DESCRIPTION
**Pull request by ghenry**

### CODE REVIEW

File: lib/src/sip_ua_helper.dart [hunk 1]
```diff
@@ -103,6 +103,7 @@ class SIPUAHelper extends EventManager {
     _settings.display_name = uaSettings.displayName;
     _settings.authorization_user = uaSettings.authorizationUser;
     _settings.user_agent = uaSettings.userAgent ?? DartSIP_C.USER_AGENT;
+    _settings.register_expires = uaSettings.register_expires;
```
Comment (cloudwebrtc) [inline lib/src/sip_ua_helper.dart:106]: Forgot to copy the `register` parameter here?
```dart
    _settings.register = uaSettings.register;
```

Comment (ghenry) [reply lib/src/sip_ua_helper.dart:106]: Oops. I'll do a new clean one?

Comment (cloudwebrtc) [reply lib/src/sip_ua_helper.dart:106]: I think just add it, and it can be merged if the test is correct.

**Comment by ghenry** [comment]
I'll do another push.

File: lib/src/sip_ua_helper.dart (hunk 1 above)
Comment (ghenry) [reply lib/src/sip_ua_helper.dart:107]: Done.

**Pull request merged**